internal/auditlog/formats.go

```diff
@@ -100,11 +102,21 @@ func (nativeFormatter) Format(al plugintypes.AuditLog) ([]byte, error) {
 			// Producer: ModSecurity for Apache/2.9.1 (http://www.modsecurity.org/).
 			// Server: Apache
 			// Engine-Mode: "ENABLED"
+
+			// AuditLogTrailer is also expected to contain the error message generated by the rule, if any
+			for _, alEntry := range al.Messages() {
+				alWithErrMsg, ok := alEntry.(auditLogWithErrMesg)
+				if ok && alWithErrMsg.ErrorMessage() != "" {
+					res.WriteByte('\n')
```

Comment:
```suggestion
					fmt.Fprintln(res, "")
```

Reply:
I can't use it as is with a string builder (complains of missing io.Writer). We are using in several places `res.WriteByte('\n')`, is it a performance or readability suggestion?

Reply:
it is because the method is in the pointer not in the value. The main use here is to make this working on windows but I see this has been used before so we can keep it.

Reply:
I see, thanks!